spring-boot-autoconfigure/src/main/java/org/axonframework/boot/autoconfig/AxonAutoConfiguration.java

```diff
@@ -179,9 +181,9 @@ public void configureEventHandling(EventHandlingConfiguration eventHandlingConfi
                 eventHandlingConfiguration.registerTrackingProcessor(k, messageSource, c -> config, sequencingPolicy);
```

Comment:
Shouldn't this guy also configure through the `eventProcessorRegistry` i.o. the `eventHandlingConfiguration`?

Reply:
If that's so, then you can also remove the `EventHandlingConfiguration` parameter from this function.

Reply:
This one is a bit trickier to change since it contains `SequencingPolicy`. And `SequencingPolicy` is important to `SimpleEventHandlerInvoker` but not the `EventProcessor`. That's why we don't have a similar method for the `EventProcessorRegistry`. The only question is whether corresponding `EventHandlingConfiguration` method should be deprecated.

Reply:
Hmmm alright, thanks for explaining that. See you've added a follow up to be able to configure the `SequencingPolicy` by the way. Good stuff